src/abstract-interpretation/data-frame/mappers/function-mapper.ts

```diff
@@ -698,7 +811,23 @@ function getEffectiveArgs(
 	args: readonly RFunctionArgument<ParentInformation>[],
 	excluded: string[]
 ): readonly RFunctionArgument<ParentInformation>[] {
-	return args.filter(arg => arg === EmptyArgument || arg.name === undefined || !excluded.includes(unescapeArgument(arg.name.content)));
+	return args.filter(arg => arg === EmptyArgument || arg.name === undefined || !excluded.includes(unquoteArgument(arg.name.content)));
+}
+
+function getEntriesFromCsvLine(line: string, sep: string = ',', quote: string = '"', comment = ''): (string | undefined)[] {
+	sep = escapeRegExp(sep, true);  // only allow tokens like `\s`, `\t`, or `\n` in separator, quote, and comment chars
+	quote = escapeRegExp(quote, true);
+	comment = escapeRegExp(comment, true);
+	const quantifier = sep === '\\s' ? '+' : '*';  // do not allow unquoted empty entries in whitespace-sparated files
+
+	const LineCommentRegex = new RegExp(`\\s*[${comment}].*`);
+	const CsvEntryRegex = new RegExp(`(?<=^|[${sep}])(?:[${quote}]((?:[^${quote}]|[${quote}]{2})*)[${quote}]|([^${sep}]${quantifier}))`, 'g');
```

Comment:
Is this the regex they use as well? i thought readr relied on a state machine. If so we may be better of using the same behavior?